### ISSUE TITLE
HCP analysis

### ISSUE THREAD

**Issue by MDijsselhof** [ExploreASL/ExploreASL#1169]
### Description

HCP data was acquired with T1, T2-FLAIR and Multi-Band EPI PCASL. The ASL data we have consists of three images:
- ASL4D
- M0 AP
- M0 PA

The ASL4D should be analysed with detail to the motion correction due to the multi-band acquisition. We will utilise the steps provided by https://github.com/physimals/hcp-asl.

**Comment by MDijsselhof** [comment]
In spm_realign, at line 161 it mentions a label-control order, is this correct?

**Comment by HenkMutsaerts** [comment]
So speaking with Matthew Glasser (PI HCP), it seems Michael's group did all the testing and pipeline adaptations already for the HCP data.

**Comment by HenkMutsaerts** [comment]
> In spm_realign, at line 161 it mentions a label-control order, saying always label first, is this correct?

Yes, it does. For motion correction of control-label series it doesn't matter if it thinks the first is label or control, also not for the zig-zag regression (I think).

@jan-petr Do you remember if the order of the data matters for the zig-zag regression?
